src/liballoc/collections/vec_deque.rs

```diff
@@ -1024,7 +1040,7 @@ impl<T> VecDeque<T> {
             iter: Iter {
                 tail: drain_tail,
                 head: drain_head,
-                ring: unsafe { self.buffer_as_mut_slice() },
+                ring: unsafe { self.buffer_as_slice() },
```

Comment:
Just to triple check - this change has nothing to do with the other changes right?

Reply:
Right, this is just a consistency change.